src/libs/actions/IOU.js

```diff
@@ -76,13 +76,16 @@ function requestMoney(report, amount, currency, payeeEmail, participant, comment
     if (chatReport.iouReportID) {
         if (isPolicyExpenseChat) {
             moneyRequestReport = {...iouReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReport.iouReportID}`]};
-            moneyRequestReport.total += amount;
+
+            // Because of the Expense reports are stored as negative values, we substract the total from the amount
+            moneyRequestReport.total = ReportUtils.isExpenseReport(moneyRequestReport) ? moneyRequestReport.total - amount : moneyRequestReport.total + amount;
         } else {
             moneyRequestReport = IOUUtils.updateIOUOwnerAndTotal(iouReports[`${ONYXKEYS.COLLECTION.REPORT}${chatReport.iouReportID}`], payeeEmail, amount, currency);
         }
     } else {
         moneyRequestReport = isPolicyExpenseChat
-            ? ReportUtils.buildOptimisticExpenseReport(chatReport.reportID, chatReport.policyID, payeeEmail, amount, currency)
+            ? // The amount for Expense reports are stored as negative value in the database
+              ReportUtils.buildOptimisticExpenseReport(chatReport.reportID, chatReport.policyID, payeeEmail, amount * -1, currency)
             : ReportUtils.buildOptimisticIOUReport(payeeEmail, payerEmail, amount, chatReport.reportID, currency);
```

Comment:
I think we also need to update the total [here](https://github.com/Expensify/App/blob/6851675645dd37a08a9c34023ac7af3695631db1/src/libs/actions/IOU.js#L1161)